regparser/tree/depth/markers.py

```diff
@@ -6,7 +6,8 @@
 from regparser.utils import roman_nums
 
 
-lower = tuple(string.ascii_lowercase)
+lower = tuple(string.ascii_lowercase) + ('aa', 'bb', 'cc', 'dd', 'ee', 'ff',
```

Comment:
This is another place where we're solving the same problem: 

https://github.com/cfpb/regulations-parser/pull/284/files#diff-232c4f6c1f52faacf64802bf1bd4996cR9